examples/website-demos/social-messaging/src/components/Thread/ThreadInner.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from 'react';
+import { Thread, useChannelStateContext } from 'stream-chat-react';
+
+import { MessageInput } from '../MessageInput/MessageInput';
+
+import { useOverrideSubmit } from '../../hooks/useOverrideSubmit';
+
+export const ThreadInner = () => {
+  const [checked, setChecked] = useState(false);
```

Comment:
do the social designs call for a send in channel option? if not we don't need the checked state, this was part of the virtual event app designs